vertx-web/src/main/java/io/vertx/ext/web/handler/AuthenticationHandler.java

```diff
@@ -66,4 +66,15 @@
   default String authenticateHeader(RoutingContext context) {
     return null;
   }
+
+  /**
+   * This method is called to perform any post authentication tasks, such as redirects.
+   * By default it calls: context.next()
```

Comment:
Sorry, my bad here, it should say that "overrides **must** call context.next()" on success. Otherwise implementations may forget to do that and will just hang.

Reply:
OK, got it :)